packages/playground/src/twoslashInlays.ts

```diff
@@ -31,12 +39,15 @@ export const createTwoslashInlayProvider = (sandbox: Sandbox) => {
           // @ts-ignore
           kind: 0,
           position: new sandbox.monaco.Position(endPos.lineNumber, endPos.column + 1),
-          text,
-          whitespaceBefore: true,
+          label: text,
+          paddingLeft: true,
```

Comment:
this commit has changed `whitespaceBefore` to `paddingLeft`:
https://github.com/microsoft/vscode/commit/8ef8933a1af8bc3613dcc375d7483c2fd6da6add